lib/styles/settings/_colors.scss

```diff
@@ -101,16 +101,6 @@ $color-fail: $color-outrageous-orange;
 $color-fail-supporting: $color-melon-orange;
 $color-fail-background: $color-ice-orange;
 
-$color-shadow: rgba($color-firefly-black, 0.35);
```

Comment:
did you think about consequences of this change? Is it backwards compatible? Do you need to adapt platform code?

Reply:
@kvas-damian I moved it to `settings/_shadows.scss` I just decided to move all shadow settings to one file.

Reply:
I understand that, but I'm worried if it won't break platform when you install this version.